TextInputKit/TextInputKit/Code/SpecializedTextInput/CardNumber/BankCardNumberUtils.swift

```diff
@@ -10,119 +10,64 @@ import Foundation
 
 struct BankCardNumberUtils {
 
-    struct IinRangeInfo {
-
-        let range: CountableClosedRange<Int>
-
-        let cardBrand: BankCardBrand
-
-        let maxCardNumberLength: Int
+    private init() {}
```

Comment:
I'n not sure this is a good idea to use an empty structure type as a namespace for methods.